apps/mobile/src/libs/components/slider-content/slider-content.tsx

```diff
@@ -1,34 +1,44 @@
 import React from "react";
 
 import { GradientSlider, Text, View } from "~/libs/components/components";
-import { type colorToGradientColors } from "~/libs/maps/maps";
+import { categoryToColors, type colorToGradientColors } from "~/libs/maps/maps";
 import { globalStyles } from "~/libs/styles/styles";
+import { type SliderData } from "~/libs/types/types";
 
 type Properties = {
-	color: keyof typeof colorToGradientColors;
-	label: string;
+	data: SliderData;
+	onValueChange: (id: number, sliderValue: number) => void;
 };
 
 const MAX_SLIDER_VALUE = 10;
-const MIN_SLIDER_VALUE = 0;
 
-const SliderContent: React.FC<Properties> = ({ color, label }) => {
+const getGradientColorsForCategory = (
+	categoryName: string,
+): keyof typeof colorToGradientColors => {
+	return categoryToColors[categoryName.replaceAll(/\s+/g, "")] ?? "blue";
```

Comment:
Move "blue" to a constant, it's not clear why blue

Reply:
delete unnecessary condition